caffeinated/admin/new/pricing/espresso_events_Pricing_Hooks.class.php

```diff
@@ -1539,7 +1539,7 @@ protected function _get_ticket_row(
                     )
                 ),
             'TKT_ID'                        => $default ? 0 : $ticket->ID(),
-            'TKT_description'               => $default ? '' : $ticket->get_f('TKT_description'),
+            'TKT_description'               => $default ? '' : $ticket->get_raw('TKT_description'),
```

Comment:
was thinking the issue is escaping on one of the admin page templates that's too strict, 
but if this works, then that's ok with me